### PR TITLE
String Rotation

### PR DESCRIPTION
**Pull request by divyaachoudharyy**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/smv1999/CompetitiveProgrammingQuestionBank/blob/master/CONTRIBUTING.md)?

Yes

### Description

Checking if string one is  rotation of another string. 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.

## Related Issues or Pull Requests
Linked Issues #572 

## Fill up the DCP form 

If you are a part of DCP 21, fill up this [form](https://docs.google.com/forms/d/e/1FAIpQLSeRAJnHE4eAa_W0IUeNZgheiy5POX3I3tQB7DtFsiB13D5w2g/viewform) after the PR is merged.

### The links provided in the form will be checked thoroughly, if someone tries fooling around, strict actions will be taken against the individual.

### CODE REVIEW

File: Strings/check_if_strings_are_rotation_of_eachother.cpp [hunk 1]
```diff
@@ -0,0 +1,37 @@
+/***************************************************************************
+
+A Program to check if strings are rotations of each other or not.
+
+***************************************************************************/
+
+SOLUTION (in C++):
```
Comment (smv1999) [inline Strings/check_if_strings_are_rotation_of_eachother.cpp:7]: Comment this line and also give explanation in multi-line comments.

**Review by smv1999 — CHANGES_REQUESTED** [review]

File: Strings/check_if_strings_are_rotation_of_eachother.cpp (hunk 1 above)
Comment (divyaachoudharyy) [reply Strings/check_if_strings_are_rotation_of_eachother.cpp:7]: Should I make a pull request again after I have done the necessary changes or is there any way to do the changes here itsefl?

Comment (smv1999) [reply Strings/check_if_strings_are_rotation_of_eachother.cpp:7]: Just the commit and push the changes through the same branch, it will reflect here.

Comment (divyaachoudharyy) [reply Strings/check_if_strings_are_rotation_of_eachother.cpp:7]: I have done it, just check it once.

Comment (Saviour1001) [reply Strings/check_if_strings_are_rotation_of_eachother.cpp:7]: The comments are good, just also comment the line "Solution in C++" only the code should remain uncommented.

Comment (divyaachoudharyy) [reply Strings/check_if_strings_are_rotation_of_eachother.cpp]: Done.

**Comment by Saviour1001** [comment]
Great work @divyaachoudharyy !!

**Comment by divyaachoudharyy** [comment]
Thank you @Saviour1001

**Pull request merged**